src/WebJobs.Extensions.DurableTask/EntityScheduler/Proxy/EntityClientProxy.cs

```diff
@@ -1,16 +1,17 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.ContextInterfaces;
```

Comment:
I believe we put all of our System `using` statements first, and then it goes in alphabetical order. There should be a warning for this.